app/models/simulation.rb

```diff
@@ -7,4 +7,7 @@ class Simulation < ActiveRecord::Base
   has_many :content, class_name: 'Content'
 
   accepts_nested_attributes_for :users
+
+  has_one :name
+  has_one :requester
```

Comment:
For these, We'll need a db column for name, not an association. Also, if we want a requester association we'll have to build a user model.